appengine/ndb/async/app_async_test.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import app_async
+import pytest
+import webtest
+
+
+@pytest.fixture
+def app(testbed):
+    return webtest.TestApp(app_async.app)
+
+
+def test_main(app, testbed):
+    app_async.Account(id='123', view_counter=4).put()
+
+    # Log the user in
+    testbed.setup_env(
```

Comment:
We have a login fixture: https://github.com/GoogleCloudPlatform/python-repo-tools/blob/master/gcp/testing/appengine.py#L113